README.md

```diff
@@ -7,14 +7,149 @@
 [![OpenSSF Scorecard](https://api.scorecard.dev/projects/github.com/solarwinds/apm-php/badge)](https://scorecard.dev/viewer/?uri=github.com/solarwinds/apm-php)
 ![GitHub License](https://img.shields.io/github/license/solarwinds/apm-php)
 
+---
 
-This repo holds the source code for the OpenTelemetry-based SolarWinds APM PHP library and its dependencies.
+SolarWinds APM PHP is an OpenTelemetry-based library for distributed tracing and observability in PHP applications. It provides automatic and manual instrumentation, seamless integration with the SolarWinds Observability platform, and supports modern PHP frameworks.
 
-| Stability level |
-|-----------------|
-| dev             |
+## Prerequisites
 
+Before you begin, ensure you have:
+- [PHP 8.1+](https://www.php.net/)
+- [PECL](https://pecl.php.net/)
+- [Composer](https://getcomposer.org/)
+
+Check your versions:
+```bash
+php -v
+composer -v
+```
+
+## Installation
+
+Install the SolarWinds APM library:
+```bash
+composer require solarwinds/apm
+```
+
+Set your service key (via environment variable):
+```bash
+export SW_APM_SERVICE_KEY=<your-service-key>
+```
+
+(Optional) Change the [default collector](apm.collector.na-01.cloud.solarwinds.com):
+```bash
+export SW_APM_COLLECTOR=<your-collector-url>
+```
+
+Install a PSR-compatible HTTP client (required for OTLP exporter):
+```bash
+composer require guzzlehttp/guzzle
+```
+
+## Example Application
+
+This section demonstrates automatic instrumentation using Slim and SolarWinds APM.
+
+### 1. Create a minimal Slim app
+```bash
+composer init --no-interaction --require slim/slim:"^4" --require slim/psr7:"^1"
+composer update
+```
+
+Create `index.php`:
+```php
+<?php
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Slim\Factory\AppFactory;
+
+require __DIR__ . '/vendor/autoload.php';
+
+$app = AppFactory::create();
+
+$app->get('/rolldice', function (Request $request, Response $response) {
+    $result = random_int(1,6);
+    $response->getBody()->write(strval($result));
+    return $response;
+});
+
+$app->run();
+```
+
+Run the app:
+```bash
+php -S localhost:8080
+```
+Visit [http://localhost:8080/rolldice](http://localhost:8080/rolldice) to test.
+
+### 2. Add zero-code instrumentation
+
+Install the OpenTelemetry PHP extension ([instructions](https://opentelemetry.io/docs/zero-code/php/#install-the-opentelemetry-extension)) and verify:
+```bash
+php --ri opentelemetry
+```
+
+Add required dependencies, more instrumentation libraries can be found [here](https://packagist.org/packages/open-telemetry/opentelemetry-sqlcommenter?query=open--telemetry%2Fopentelemetry-):
```

Comment:
Nit: maybe a slight reword to emphasize this is APM and not just vanilla OTel PHP. Can take or leave!

```suggestion
Add SolarWinds APM Library and required dependencies, more instrumentation libraries can be found [here](https://packagist.org/packages/open-telemetry/opentelemetry-sqlcommenter?query=open--telemetry%2Fopentelemetry-):
```